views/js/qtiCreator/helper/languages.js

```diff
@@ -87,10 +87,20 @@ define(['util/url', 'core/dataProvider/request'], function (urlUtil, request) {
         });
     };
 
+    /**
+     * Does language support vertical writing mode
+     * @param {String} code language code ex: 'ar-arb'
+     * @returns {boolean}
+     */
+    const supportsVerticalWritingMode = code => {
+        return code && code.startsWith('ja-');
```

Comment:
Ideally this should be added to backend endpoint (`https://construct1-oat-unit01.dev.gcp-eu.taocloud.org/tao/Languages/index`) - same as for `isRTLbyLanguageCode`.

But maybe it's fine to simplify and hardcode it?

PS: Languages with `vertical-rl` are _Japanese, Chinese, Korean_. Language with `vertical-lr` is _Mongolian_.
https://www.w3.org/International/articles/vertical-text/
https://en.wikipedia.org/wiki/Horizontal_and_vertical_writing_in_East_Asian_scripts

Reply:
Changed to this backend solution in https://github.com/oat-sa/extension-tao-itemqti/pull/2661/commits/75e254ebbba64838e38749af7b5be352f8a26503
BE: https://github.com/oat-sa/tao-core/pull/4168

Let's go back to FE-only if you think it's a needless complication.